fish-pepper/run-java-sh/fp-files/container-limits

```diff
@@ -39,12 +39,13 @@ core_limit() {
 max_memory() {
   # High number which is the max limit unti which memory is supposed to be
   # unbounded. 
-  local max_mem_unbounded="$(cat /sys/fs/cgroup/memory/memory.memsw.limit_in_bytes)"
   local mem_file="/sys/fs/cgroup/memory/memory.limit_in_bytes"
   if [ -r "${mem_file}" ]; then
-    local max_mem="$(cat ${mem_file})"
-    if [ ${max_mem} -lt ${max_mem_unbounded} ]; then
-      echo "${max_mem}"
+    local max_mem_cgroup="$(cat ${mem_file})"
+    local max_mem_meminfo=$(($(awk '/MemTotal/ {print $2}' /proc/meminfo)*1024))
```

Comment:
Can we change this too 

```
local max_mem_cgroup=$(cat /proc/meminfo | awk '/MemTotal/ {printf "%u", $2 * 1024}')
```

This would avoid the usage of the bashism `$(( ...))` so that this script still might work for other shells, too.